packages_rs/nextclade-cli/src/cli/nextclade_cli.rs

```diff
@@ -494,10 +495,28 @@ pub struct NextcladeRunOutputArgs {
   /// If the provided file path ends with one of the supported extensions: "gz", "bz2", "xz", "zstd", then the file will be written compressed. Use "-" to write the uncompressed to standard output (stdout).
   ///
   /// If the required directory tree does not exist, it will be created.
-  #[clap(long, short = 't')]
+  #[clap(long, short = 't', group = "tabular_output")]
   #[clap(value_hint = ValueHint::AnyPath)]
   pub output_tsv: Option<PathBuf>,
 
+  /// Restricts columns written into tabular output files (CSV and TSV).
+  ///
+  /// Should contain a comma-separated list of individual column names and/or column category names to include into both CSV and TSV outputs.
+  ///
+  /// If this flag is omitted, or if category 'all' is present in the list, then all other entries are ignored and all columns are written.
+  ///
+  /// Only valid together with one or multiple of flags: `--output-csv`, `--output-tsv`, `--output-all`.
+  #[clap(
+    long,
+    short = 'C',
+    takes_value = true,
+    multiple_values = true,
+    use_value_delimiter = true
+  )]
+  #[clap(requires = "tabular_output")]
+  #[clap(possible_values=CSV_POSSIBLE_CATEGORIES.iter().map(String::as_str))]
```

Comment:
We need to add all individual column names here, too

Reply:
See https://github.com/nextstrain/nextclade/tree/fix-individual-csv-columns for a (partial?) fix